File: programación/Momento2/5.15/max_min.py
i = 1 
numero = int(input(f"{i}: Ingresa un número: "))
maxn = numero
minn = numero

for i in range(2, 101):
    numero = int(input(f"{i}: Ingresa un número: "))
    if numero > maxn:
        maxn = numero
    elif numero < minn:
        minn = numero

print(f"Maximo: {maxn}, Minimo: {minn}")

File: programación/Momento2/5.15/max_min.c
#include "stdio.h"

int main(void){
    int i;
    float numero, max, min;
    i = 1;
    printf("(%d) Ingresa un número:\n", i);
    scanf("%f", &numero);
    max = numero;
    min = numero;
    for (int i = 2; i < 101; i++) {
        printf("(%d) Ingresa un número:\n", i);
        scanf("%f", &numero);
        if (numero > max) {
            max = numero;
        } else {
            if (numero < min) {
                min = numero;
            }
        }
    }
    printf("Máximo %f Minimo %f\n", max, min);
}